ikenna-portfolio/portfolio/bin/deploy_site

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This is really good. Can you sharing how you do the automation next week when it's your turn for tech sharing?

Reply:
will do